### PR TITLE
some custom domain fixes

### PR DESCRIPTION
**Pull request by ACoderLife**
There was a few small issues with the Vnext branch when you use your own domain like this:

npm run deploy --email=my@email.com --domain=custom.com --hostedzone=A066415336KDJALFWFI77

They were failing the deployment.

The admin and application sites have stopped working due to build failures within CodeBuild due to over updated angular-cli "^versioning".  This pull request pins the version to a known working verison.

### CODE REVIEW

**Comment by ranjith-aws** [comment]
Hi @ACoderLife - Thanks for the PR. We will review this and get back if we need any additional details. @zpbappi-aws

File: lib/api-stack.ts [hunk 1]
```diff
@@ -165,14 +165,15 @@ export class ApiStack extends Stack {
...
                     }
                 ]
-            });
+            }); */
 
             new route53.ARecord(this, 'CustomDomainAliasRecord', {
                 zone: publicHostedZone!,
-                target: route53.RecordTarget.fromAlias(new targets.ApiGatewayv2DomainProperties(
-                    domain.domainName,
+                target: route53.RecordTarget.fromAlias(new targets.ApiGateway(api)
```
Comment (zpbappi-aws) [inline lib/api-stack.ts:172]: Good pick. Can you please remove the commented code?

File: lib/api-stack.ts [hunk 2]
```diff
@@ -155,7 +155,7 @@ export class ApiStack extends Stack {
         if (useCustomDomain) {
             
 
-            const domain = new apigwv2.CfnDomainName(this, "SaaSApiCustomDomain", {
+            /* const domain = new apigwv2.CfnDomainName(this, "SaaSApiCustomDomain", {
```
Comment (zpbappi-aws) [inline lib/api-stack.ts:158]: Please remove the commented code.

File: clients/Admin/package.json [hunk 3]
```diff
@@ -44,10 +44,10 @@
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^12.0.4",
-    "@angular/cli": "^12.0.4",
-    "@angular/compiler-cli": "^12.0.4",
-    "@angular/language-service": "^12.0.4",
+    "@angular-devkit/build-angular": "12.2.16",
+    "@angular/cli": "12.2.16",
```
Comment (zpbappi-aws) [inline clients/Admin/package.json:48]: Thank you. There seems to be a breaking change. Can you please make the exact changes in the `Landing/package.json` as well?

**Review by zpbappi-aws — CHANGES_REQUESTED** [review]

File: clients/Admin/package.json (hunk 3 above)
Comment (ACoderLife) [reply clients/Admin/package.json:48]: The problem is not in Landing since it is not using coreui.  I have updated landing anyway to be consistent.

**Comment by ACoderLife** [comment]
Done 😊

**Comment by zpbappi-aws** [comment]
Thank you for your contribution, @ACoderLife

**Pull request merged**